pkg/codegen/internal/test/testdata/aws-fargate-pp/nodejs/aws-fargate.ts

```diff
@@ -4,9 +4,9 @@ import * as aws from "@pulumi/aws";
 const vpc = aws.ec2.getVpc({
     "default": true,
 });
-const subnets = vpc.then(vpc => aws.ec2.getSubnetIds({
-    vpcId: vpc.id,
-}));
+const subnets = aws.ec2.getSubnetIdsOutput({
+    vpcId: vpc.then(vpc => vpc.id),
+});
```

Comment:
This is interesting--so it looks like we're choosing the output form if there are any eventuals in the args, not just `Output`s. IMO we should adjust that so that we only choose the output form if there are `Output`s in the args (consistent with my general opinion that we should only use `Output<>` in our examples where we must).

Reply:
Hm okay, I think I see. So it should continue lifting over promises but not over outputs. I can work this out tomorrow morning.

Reply:
This one is done.